clang/test/3C/graphs.c

```diff
@@ -267,12 +265,12 @@ struct node* createNode(int v)
 /*Allocate memory for the entire graph structure*/
 
 struct Graph* createGraph(int vertices)
-	//CHECK: _Ptr<struct Graph> createGraph(int vertices)
+	//CHECK: _Ptr<struct Graph>  createGraph(int vertices)
 
```

Comment:
I don't know if spaces effect the correctness here, but this one, and similar ones above, seem like they should not be added, whereas the deleted spaces elsewhere in the file do seem better.